plugins/SitesManager/Controller.php

```diff
@@ -73,6 +79,11 @@ public function getGlobalSettings()
         $globalSettings['excludedQueryParametersGlobal'] = API::getInstance()->getExcludedQueryParametersGlobal();
         $globalSettings['excludedUserAgentsGlobal'] = API::getInstance()->getExcludedUserAgentsGlobal();
         $globalSettings['excludedReferrersGlobal'] = API::getInstance()->getExcludedReferrersGlobal();
+        $globalSettings['exclusionTypeForQueryParams'] = API::getInstance()->getExclusionTypeForQueryParams();
+
+        if ($globalSettings['exclusionTypeForQueryParams'] !== 'custom_exclusions') {
```

Comment:
```suggestion
        if ($globalSettings['exclusionTypeForQueryParams'] !== SitesManager::URL_PARAM_EXCLUSION_TYPE_NAME_CUSTOM_EXCLUSIONS) {
```

We should prefer the constant if we have it.

Though I am wondering why we need that check here, and not also in another place where the query parameters are accessed:

https://github.com/matomo-org/matomo/blob/c5d3fad1ce8c56a51f4150db7437c6e5b2d95afb/plugins/SitesManager/SitesManager.php#L335-L342

Is there any special case where these two areas should differ in a way that cannot be managed inside the API methods?

Reply:
Good point, the controller should be dumb, so I've removed that check. The API handles this already. Take a look and let me know if that's what you are thinking, and you can resolve the thread.